README.md

```diff
@@ -208,6 +202,24 @@ input {
   }
 ```
 
+#### Cross tenant function
+
+The output plugin is able to post data on behalf of a tenant to the monasca-log-api.
+For example you can use a user of tenant X to send logs to the monasca-log-api for another tenant Y. The logs will be stored for the tenant Y. And only users of tenant Y will see the logs.
```

Comment:
"The logs will be stored in the index of tenant Y." I would delete the last sentence.